src/main/resources/banner.txt

```diff
@@ -0,0 +1,5 @@
+   ___                      __               __  ____
+  / _ | ___________ _    __/ /  ___ ___ ____/ / / __/
+ / __ |/ __/ __/ _ \ |/|/ / _ \/ -_) _ `/ _  / /__ \ 
+/_/ |_/_/ /_/  \___/__,__/_//_/\__/\_,_/\_,_/ /____/ 
```

Comment:
I would keep this for the Core/Support systems. Herwe we should have someting else.